reference/Commerce/src/Commerce/Platforms/iOS/Main.iOS.cs

```diff
@@ -9,6 +9,10 @@ public static void Main(string[] args)
     {
         // if you want to use a different Application Delegate class from "AppDelegate"
         // you can specify it here.
```

Comment:
```suggestion
```

Reply:
There's too many of these to add a comment for each one :), let's just get rid of the rest of these comments in all the other reference apps